cpp/CMakePresets.json

```diff
@@ -220,6 +220,21 @@
         "PARQUET_REQUIRE_ENCRYPTION": "ON"
       }
     },
+    {
+      "name": "features-valgrind",
+      "inherits": [
+        "features-maximal"
```

Comment:
Could you please remove this? It should be possible to build for Valgrind without enabling every Arrow component. Valgrind doesn't fail if Gandiva is disabled (!).

Reply:
The "features" presets should be considered building blocks for other presets, that's why they're hidden. It should be possible to combine e.g. "features-valgrind" and "features-minimal" for a lightweight Valgrind build.

Reply:
OK.
I've removed `features-maximal` from `features-valgrind` and added `ninja-debug-valgrind-*` that use `features-minimal`, `features-basic` and `features-maximal`.